File: swift/sprint0/E.swift
func getTwoSumFast(_ a: [Int], _ target: Int) -> (Int, Int)? {
    // Your code
    // “ヽ(´▽｀)ノ”
}

func readInts() -> [Int] {
    return readLine()!.split(separator: " ").map { (x: Substring) -> Int in
        return Int(x)!
    }
}


let size = Int(readLine()!)!
let a = readInts()
let target = Int(readLine()!)!
let pair = getTwoSumFast(a, target)
if (pair == nil) {
    print("None")
} else {
    print(pair!.0, pair!.1, terminator: " ")
}

File: kotlin/sprint0/E.kt
private fun getTwoSumFast(a: List<Int>, k: Int): Pair<Int, Int>? {
    // Your code
    // ヽ(´▽`)/
}

fun main() {
    val size = readInt()
    val a = readInts()
    val k = readInt()
    val result = getTwoSumFast(a, k)
    if (result == null) {
        println("None")
    } else {
        println("${result.first} ${result.second}")
    }
}

private fun readStr() = readln()
private fun readInt() = readStr().toInt()
private fun readStrings() = readStr().split(" ")
private fun readInts() = readStrings().map { it.toInt() }